### ISSUE TITLE
Nested Swiper within a Virtual Swiper doesn't loop correctly, some slides disappear

### ISSUE THREAD

**Issue by m345054** [nolimits4web/swiper#6966]
### Check that this is really a bug

- [X] I confirm

### Reproduction link

https://codesandbox.io/p/sandbox/swiper-virtual-slides-forked-gyk955?file=%2Findex.html%3A82%2C29

### Bug description

When using a nested Swiper (horizontal) within a virtual Swiper (vertical), the loop functionality seems to break intermittently. Specifically, some slides from the nested horizontal Swiper vanish unpredictably. Although these slides are added in the `init` method using `appendSlide`, they sometimes completely disappear from the HTML.

### Expected Behavior

All the horizontal slides should be present, regardless of the vertical slide position or interactions. Test1, Test2 and Test3 should always be present in  the horizontal swiper.

### Actual Behavior

Random horizontal slides disappear from the nested Swiper, even though they were appended using the `appendSlide` method during initialization. The behavior is intermittent and challenging to consistently reproduce, but it is evident that slides vanish.

### Swiper version

10.2.0

### Platform/Target and Browser Versions

Windows 11 Chrome

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a Swiper issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [ ] I'm willing to open a PR

**Comment by nolimits4web** [comment]
You need to disable cache and init inner swipers everytime they are added to DOM. Or try using Swiper Element instead

**Comment by marcelschmidttuk** [comment]
Thank you for the response!

I've adjusted the codesandbox as per your suggestion by disabling the cache and ensuring that inner swipers are initialized every time they're added to the DOM. Unfortunately, the issue still persists.

I've already tried using the Swiper Element in my main project, but the problem remained unchanged with that approach as well.

Would appreciate any further insights or recommendations to resolve this. Thanks again for your assistance!

**Comment by m345054** [comment]
@nolimits4web 
Thank you for the response!

I've adjusted the codesandbox as per your suggestion by disabling the cache and ensuring that inner swipers are initialized every time they're added to the DOM. Unfortunately, the issue still persists.

I've already tried using the Swiper Element in my main project, but the problem remains unchanged with that approach as well.

Would appreciate any further insights or recommendations to resolve this. Thanks again for your assistance!
